comprehensive/lms/static/sass/_overrides.scss

```diff
@@ -332,6 +332,9 @@ button#orders-tab, button#accounts-tab {
                 float: left;
                 margin: 0 20px 20px 0;
             }
+            .nav-colophon a{
```

Comment:
please don't merge until after Thursday's deployment. we'll have the proper branches in place soon so this won't be in issue shortly